source/MaterialXFormat/Util.h

```diff
@@ -0,0 +1,52 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_FORMAT_UTIL_H
+#define MATERIALX_FORMAT_UTIL_H
+
+/// @file
+/// Format utility methods
+
+#include <MaterialXCore/Document.h>
+#include <MaterialXCore/Element.h>
+#include <MaterialXCore/Interface.h>
+
+#include <MaterialXFormat/File.h>
+#include <MaterialXFormat/XmlIo.h>
+
+namespace MaterialX
+{
+
+/// Removes the extension from the provided filename
+string removeExtension(const string& filename);
+
+/// Read the given file and return a string containing its contents; if the read is not
+/// successful, then the empty string is returned.
+string readFile(const string& filename);
```

Comment:
Now that these two functions are part of MaterialXFormat, we should probably change their arguments from std::string to FilePath for consistency with other functionality in the library.

Reply:
Yes, I think these we're written before the switch to FilePath and should be updated now.